lib/chef/provider/powershell_script.rb

```diff
@@ -24,71 +24,147 @@ class PowershellScript < Chef::Provider::WindowsScript
 
       provides :powershell_script, os: "windows"
 
+      def initialize (new_resource, run_context)
+        super(new_resource, run_context, '.ps1')
+        normalize_script_exit_status
+      end
+
+      def action_run
+        valid_syntax = validate_script_syntax!
```

Comment:
Now I'm trying to understand why I even need to care about the return value. That's what happens when you leave a PR for a month or two :frowning: .

Seems like I should just let it raise an exception or keep going.